src/models/stores/unit-configuration.ts

```diff
@@ -79,4 +79,8 @@ export interface UnitConfiguration extends ProblemConfiguration {
   comparisonPlaceholderContent: string | string[];
   // configuration of navigation tabs (document navigation UI)
   navTabs: SnapshotIn<typeof NavTabsConfigModel>;
+  // used for AI tagging
+  showCommentTag?: boolean;
+  defaultTag?: string;
```

Comment:
```suggestion
  tagPrompt?: string;
```
Unless I'm misunderstanding something, this property isn't a tag at all -- it's a prompt to the user to help them understand what the tags are all about. It happens to be the case that the prompt is displayed to the user by making it the initial value of the `select`, but it could also be implemented by rendering it separately outside the `select` entirely, so calling it the `defaultTag` seems misleading. There may be a better name than `tagPrompt`, but that's the first thing that occurred to me.